media-sound/teamspeak-client-bin/teamspeak-client-bin-3.0.16.ebuild

```diff
@@ -20,6 +20,7 @@ IUSE="alsa pulseaudio"
 REQUIRED_USE="|| ( alsa pulseaudio )"
 
 RDEPEND="
+	dev-libs/quazip[qt5]
```

Comment:
I have no specific plan. All versions in tree have soname 1.0.0. But perhaps it would be beneficial for at least this package to have a subslot declared. Feel free to add it.